manim/mobject/changing.py

```diff
@@ -108,35 +124,50 @@ def construct(self):
                 self.add(trace, rolling_circle)
                 self.play(rolling_circle.animate.shift(8*RIGHT), run_time=4, rate_func=linear)
 
+    .. manim:: DissipatingPathExample
+
+        class DissipatingPathExample(Scene):
+            def construct(self):
+                path = NumberPlane().get_graph(lambda x: np.sin(x))
+                a = Dot(path.get_start())
+                b = TracedPath(a.get_center, dissipating_time=0.5, stroke_opacity=[1, 0])
+                self.add(a, b)
+                self.play(
+                    MoveAlongPath(a, path),
+                    run_time=5,
+                )
+                self.wait()
+
     """
 
+    @deprecated_params(
+        params="min_distance_to_new_point", since="v0.10.0", until="v0.11.0"
```

Comment:
Deprecation period is 2 releases, unless that has changed..

Reply:
Ah, I didn't know that. I looked at [this](https://github.com/ManimCommunity/manim/blob/main/manim/mobject/three_dimensions.py#L79-L84) and saw deprecation period was 1 release, so I got confused.